client/extensions/zoninator/components/forms/zone-content-form/posts-list.jsx

```diff
@@ -79,25 +79,26 @@ class PostsList extends Component {
 					</SearchAutocomplete>
 				</FormFieldset>
 
-				{ !! posts.length &&
-					! requesting && (
-						<FormFieldset>
-							<p className={ explanationTextClass }>
-								{ translate(
-									"You can reorder the zone's content by dragging it to a different location on the list."
-								) }
-							</p>
-							<SortableList direction="vertical" onChange={ this.changePostOrder( fields ) }>
-								{ posts.map( ( post, index ) => (
-									<PostCard
-										key={ index }
-										post={ post }
-										remove={ this.removePost( fields, index ) }
-									/>
-								) ) }
-							</SortableList>
-						</FormFieldset>
-					) }
+				{ !! posts.length && ! requesting && (
+					<FormFieldset>
+						<p className={ explanationTextClass }>
+							{ translate(
+								"You can reorder the zone's content by dragging it to a different location on the list."
+							) }
+						</p>
+						<SortableList direction="vertical" onChange={ this.changePostOrder( fields ) }>
+							{ posts.map( ( post, index ) => (
+								<PostCard
+									key={ index }
```

Comment:
Can we use a non-index while we're here? `post.id` should be fine I think.

Rationale: https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318